sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -291,11 +292,10 @@ public partial class KeyVaultSelectiveKeyRestoreResult
     public partial class KeyVaultSetting
     {
         public KeyVaultSetting(string name, bool value) { }
+        public string Content { get { throw null; } }
         public string Name { get { throw null; } }
-        public Azure.Security.KeyVault.Administration.SettingType? Type { get { throw null; } }
-        public bool AsBoolean() { throw null; }
-        public string AsString() { throw null; }
-        public override string ToString() { throw null; }
+        public Azure.Security.KeyVault.Administration.KeyVaultSettingType? SettingType { get { throw null; } }
```

Comment:
I find it inconsistent that this property is called SettingType but the one below is not called SettingValue.

Reply:
IMO, "SettingType" is already duplicative given the type is "KeyVaultSetting", but I appreciate we don't want properties called just "Type". That said, it seems even more extraneous to call the value "SettingValue".

Reply:
Yes, I agree, this is a concession to avoid a property called just `Type`.  I prefer `Name` and `Value` for the other two properties and would relent on using `Type` before making it `SettingValue`.